src/MetricKit/MXMetaData.cs

```diff
@@ -0,0 +1,21 @@
+
+using System;
+
+using Foundation;
+using ObjCRuntime;
+using UIKit;
+
+namespace MetricKit {
+
+	public partial class MXMetaData {
+
+		public virtual NSDictionary DictionaryRepresentation {
+			get {
+				if (PlatformHelper.CheckSystemVersion (14,0))
```

Comment:
This version check doesn't work on macOS nor watchOS. This framework might not be on neither platform right now, but I think we should consider the possibility that it might be in the future, and do something like:

```suggestion
#if __IOS__
				if (PlatformHelper.CheckSystemVersion (14, 0))
#else
#error Platform-specific logic not implemented
#endif
```

Reply:
It's an iOS only framework but it can't hurt :)